src/module/common/component/display/StatusIcon/StatusIcons.tsx

```diff
@@ -0,0 +1,55 @@
+import { IconButton, IconButtonProps, ThemeIcons, ThemePalette, useTheme } from "@peersyst/react-native-components";
+
+export interface StatusIconsProps extends Omit<IconButtonProps, "children"> {
+    statusIcon: "info" | "warning" | "error" | "success" | "valid" | "invalid";
+    statusColor?: keyof ThemePalette["status"];
```

Comment:
Realment s'utilitza el statusColor? Vull dir pot ser que un valid tingui statusColor error?

Reply:
Bueno era x si vols fer in info amb color warning o valid x ejemple.. Lo seu hagués sigut posar la prop de color al Icon de la lib

Reply:
Sí, és una pending task